packages/vue/src/components/index.ts

```diff
@@ -13,13 +13,13 @@ export {
     Operation,
 } from "./FCrudDataset";
 export * from "./FDataTable";
-export * from "./FDatepickerField";
+export { FDatepickerField } from "./FDatepickerField";
 export * from "./FErrorList";
 export * from "./FExpand";
 export * from "./FExpandablePanel";
 export * from "./FExpandableParagraph";
-export * from "./FFieldset";
-export * from "./FFileItem";
+export { type FFieldsetAPI, FFieldset, useFieldset } from "./FFieldset";
```

Comment:
What is your opinion on exporting `useFieldset`? Should we consider it internal or external? Or rather, do we expect consumers to use `useFieldset()` for anything themselves?

Reply:
If we don't see any use for it right now, then skip it :
removing it later -> breaking change.
adding it when needed -> not a breaking change.

Reply:
Removed export